wiktor_hardware_interface/JointValueQuantity joint_stiffness
wiktor_hardware_interface/JointValueQuantity joint_damping
